sous-chef/sous_chef/menu/create_menu/create_menu.py

```diff
@@ -15,11 +20,34 @@
 
 
 @dataclass
-class Menu(MenuForGroceryList, MenuFromFixedTemplate):
-    def upload_menu_to_todoist(self, todoist_helper: TodoistHelper):
+class Menu(MenuFromFixedTemplate):
+    def temporarily_output_menu_for_review(self):
+        pass
+
+    def finalize_menu_to_external_services(self):
```

Comment:
- [ ] @ArBridgeman better in integration tests to test via this function than `test__upload_menu_to_todoist`

Reply:
@AkagiShigeru this should not be a blocker as more changes will be made in subsequent PRs. This is just if I have time prior to when you have time to review & a reminder to myself.